src/components/form-textarea/form-textarea.js

```diff
@@ -97,36 +97,39 @@ export default {
         return null
       }
 
-      // Remember old height (includes `px` units) and reset it temporarily to `auto`
-      const oldHeight = el.style.height
-      el.style.height = 'auto'
-
       // Get current computed styles
       const computedStyle = getCS(el)
       // Height of one line of text in px
       const lineHeight = parseFloat(computedStyle.lineHeight)
-      // Minimum height for min rows (browser dependant)
-      const minHeight = parseInt(computedStyle.height, 10) || lineHeight * this.computedMinRows
-      // Calculate height of content
-      const offset =
+      // calculate height of border and padding
+      const border = (
         (parseFloat(computedStyle.borderTopWidth) || 0) +
-        (parseFloat(computedStyle.borderBottomWidth) || 0) +
+        (parseFloat(computedStyle.borderBottomWidth) || 0)
+      )
+      const padding = (
         (parseFloat(computedStyle.paddingTop) || 0) +
         (parseFloat(computedStyle.paddingBottom) || 0)
-      // Calculate content height in "rows"
-      const contentRows = Math.max((el.scrollHeight - offset) / lineHeight, 2)
+      )
+      // calculate offset
+      const offset = (computedStyle.boxSizing === 'border-box' ? border + padding : 0 )
```

Comment:
Actually, Bootstrap CSS always has everything in `border-box` mode... so this test isn't really needed.

Reply:
https://github.com/twbs/bootstrap/blob/c6c9e322f7a30d380dbcbef77f31ced85df790c8/scss/_reboot.scss#L11-L23

```scss
// Document
//
// 1. Change from `box-sizing: content-box` so that `width` is not affected by `padding` or `border`.
// 2. Change the default font family in all browsers.
// 3. Correct the line height in all browsers.
// 4. Prevent adjustments of font size after orientation changes in IE on Windows Phone and in iOS.
// 5. Change the default tap highlight to be completely transparent in iOS.


*,
*::before,
*::after {
  box-sizing: border-box; // 1
}
```

Reply:
As well, brackets are not needed around this computation